src/lib/components/BlogLink.svelte

```diff
@@ -0,0 +1,87 @@
+<script lang="ts">
+  import { formatDate } from "$lib/utils/services/datefmt";
+  import type { EndPoints } from "$lib/utils/types/microcms";
+
+  export let blogData: EndPoints["get"]["blogs"];
+  const { id, title, publishedAt, tags, ogpImg } = blogData;
+</script>
+
+<div>
+  <div class="ogp">
+    <a href={`/blogs/${id}`}>
+      {#if ogpImg}
+        <img
+          src={ogpImg.url}
+          alt={title}
+          height={ogpImg.height}
+          width={ogpImg.width}
+        />
+      {:else}
+        <img src="https://placehold.jp/1200x630.png" alt="ogp" />
+      {/if}
+    </a>
+  </div>
```

Comment:
これは `Open Graph Protocol` のことならば OGP ではないから `thumbnail` とかの方がいいかも...?
`Open Graph Protocol` でなくても何を指すか分かりやすい名前の方がいい

```suggestion
  <div class="thumbnail">
    <a href={`/blogs/${id}`}>
      {#if ogpImg}
        <img
          src={ogpImg.url}
          alt={title}
          height={ogpImg.height}
          width={ogpImg.width}
        />
      {:else}
        <img src="https://placehold.jp/1200x630.png" alt="thumbnail" />
      {/if}
    </a>
  </div>
```